guides/common/modules/proc_enabling-the-project-client-name-repository-for-rhel-7-and-rhel-6.adoc

```diff
@@ -0,0 +1,59 @@
+[id="enabling-the-project-client-name-repository-rhel-7-and-rhel-6"]
+= Enabling the {project-client-name} repository for {RHEL} 7 and {RHEL} 6
+
+[NOTE]
+====
+You require *{RHEL} Extended Lifecycle Support (ELS) Add-on* subscription to enable the repositories of {RHEL} 6.
+For more information, see https://www.redhat.com/en/resources/els-datasheet[{RHEL} Extended Lifecycle Support (ELS) Add-on] guide.
+====
+
+To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
+
+* xref:CLI_Enabling_the_Client_Repository_rhel_7_{context}[]
+* xref:CLI_Enabling_the_Client_Repository_rhel_6_{context}[]
+
+ifeval::["{mode}" == "disconnected"]
+.Prerequisites
+* Ensure that you import all content ISO images that you require into {ProjectServer}.
+endif::[]
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Red Hat Repositories*.
+. In the *Available Repositories* pane, enable the *Recommended Repositories* to get the list of repositories.
+. In the *Available Repositories* pane, click on *{project-client-name} (for RHEL 7 Server) (RPMs)* or *{project-client-name} (for RHEL 6 Server - ELS) (RPMs)* to expand the repository set.
++
+If the *{project-client-name}* items are not visible, it may be because they are not included in the Red{nbsp}Hat Subscription Manifest obtained from the Customer Portal.
+To correct that, log in to the Customer Portal, add these repositories, download the Red{nbsp}Hat Subscription Manifest and import it into {Project}.
```

Comment:
alternative:
```suggestion
Ensure that the *{project-client-name}* are part of your Red{nbsp}Hat Subscription Manifest obtained from the Customer Portal.
```

for capitalization, see https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization

Reply:
I updated capitalization for "RH subscription manifest", but I don't feel comfortable making a change like this without re-testing the steps (and re-testing is out of scope of this PR, as explained in its description).